### PR TITLE
Remove backfill

### PR DESCRIPTION
**Pull request by raoulsullivan**
This ensures we can translate back from a finance_application_v3.

### CODE REVIEW

File: sme_finance_application_schema/translations.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import re
 
-def finance_application_v3_to_sme_contact_v3(finance_application):
+def finance_application_v3_to_sme_contact_v3(finance_application, backfill_value='Unknown'):
```
Comment (thejameshayes) [inline sme_finance_application_schema/translations.py]: I don't think this should be a param as 'Unknown' is currently a static value in the code.

File: sme_finance_application_schema/translations.py [hunk 2]
```diff
@@ -12,6 +12,14 @@ def finance_application_v3_to_sme_contact_v3(finance_application):
         'telephone': applicant.get('telephone'),
         'company_number': requesting_entity.get('company_number'),
     }
+    # These cannot be 'None' in finance_application_v3
+    # so a backfill value may have been used to populate them.
+    # This value should be removed at this point
+    properties_that_could_have_been_backfilled = ['sme_name', 'applicant_surname', 'applicant_first_name']
+    for prop in properties_that_could_have_been_backfilled:
+        if sme_contact_v3[prop] == backfill_value:
+            sme_contact_v3[prop] = None
```
Comment (thejameshayes) [inline sme_finance_application_schema/translations.py]: Should the attr be popped instead of setting it to none?

Comment (thejameshayes) [reply sme_finance_application_schema/translations.py]: nwm I've found `_remove_key_if_value_is_none`

File: sme_finance_application_schema/translations.py (hunk 1 above)
Comment (raoulsullivan) [reply sme_finance_application_schema/translations.py]: *nod*

**Pull request merged**